packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -199,117 +280,130 @@ const AppNavigation = ({ items, primaryContent }: Props) => {
-                                                return {
-                                                    key: id,
-                                                    callback: () => {
-                                                        item.callback();
-                                                        return true;
-                                                    },
-                                                    label: title,
-                                                };
-                                            }),
-                                        },
-                                    ]}
-                                />
-                            ) : undefined}
-                        </SecondaryMenu>
-                    </Secondary>
-                </>
+                <InnerWrap>
+                    <KeepWidth border={!sticky}>
+                        <Primary ref={primary}>
+                            {primaryContent ||
+                                itemsPrimary.map(item => {
+                                    const { id, title } = item;
+                                    const active = isRouteActive(routeName, id);
+                                    return (
+                                        <StyledNavLink
+                                            key={id}
+                                            active={active}
+                                            onClick={item.callback}
+                                            {...(item['data-test'] && {
+                                                'data-test': item['data-test'],
+                                            })}
+                                        >
+                                            {item.icon && (
+                                                <IconWrapper>
+                                                    <Icon
+                                                        size={18}
+                                                        icon={item.icon}
+                                                        color={
+                                                            active
+                                                                ? theme.TYPE_DARK_GREY
+                                                                : undefined
+                                                        }
+                                                    />
+                                                </IconWrapper>
+                                            )}
+
+                                            <Text>{title}</Text>
+                                        </StyledNavLink>
+                                    );
+                                })}
+                        </Primary>
+                        <Secondary ref={secondary}>
+                            {condensedSecondaryMenuVisible && (
+                                <SecondaryMenuCondensed>
```

Comment:
screen width between 768px and 832px 

SecondaryMenuCondensed background is weird and the position of the opened dropdown is also incorrect
![image](https://user-images.githubusercontent.com/3729633/124740723-56e6a800-df1b-11eb-80d6-41093c75af51.png)

Reply:
added padding and fixed positioning in 331a45e053a53457e46506ccff18f29dd5f5b5fc

Reply:
this is what I meant by the weird background - white box (with gradient) above the three dots. It's maybe hard to see on some monitors, but it's there. To me it looks like a leftover from some old design. On production, this background is there but full-height.
![whitebox](https://user-images.githubusercontent.com/3729633/124786619-279a6000-df48-11eb-9bd6-46d80037153a.png)

Reply:
8db5d6e6ab43e33341df83c52e00b2e0c733565e